config/rosco.yml

```diff
@@ -8,6 +8,17 @@ redis:
 aws:
   enabled: ${providers.aws.enabled:false}
 
+azure:
+  enabled: ${providers.azure.enabled:false}
+  accounts:
+    - name: ${providers.azure.primaryCredentials.name}
+      clientId: ${providers.azure.primaryCredentials.clientId}
+      appKey: ${providers.azure.primaryCredentials.appKey}
+      tenantId: ${providers.azure.primaryCredentials.tenantId}
+      subscriptionId: ${providers.azure.primaryCredentials.subscriptionId}
+      packerResourceGroup: ${providers.azure.primaryCredentials.packerResourceGroup}
+      storageAccount: ${providers.azure.primaryCredentials.packerStorageAccount}
```

Comment:
This looks to be `packerStorageAccount` in rosco: https://github.com/spinnaker/rosco/blob/c77f2fb254dae506e88b99fc00db6d3a26996927/rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/azure/config/RoscoAzureConfiguration.groovy#L88